doc/conf.py

```diff
@@ -35,37 +41,37 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
-    'sphinx.ext.githubpages',
+    "sphinx.ext.githubpages",
 ]
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ["_templates"]
 
 # The suffix of source filenames.
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = ".rst"
 
 # The encoding of source files.
-source_encoding = 'utf-8'
+source_encoding = "utf-8"
 
 # The master toctree document.
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
-project = 'Spyder'
-copyright = " 2018 Spyder Doc Contributors <a href='https://opensource.org/licenses/MIT' target='_blank'>MIT License</a>"
-author = 'The Spyder Doc Contributors'
+project = "Spyder"
+copyright = " 2020 Spyder Doc Contributors "
```

Comment:
Do we have an easy way to embed a link to a license and/or legal notices in the footer (or somewhere else on all pages)? We're going to need it.

Reply:
Yep

Reply:
Thanks! I'll update it further after the theme is merged and stable.